sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/BasicRelationship.java

```diff
@@ -93,8 +93,8 @@ public String getRelationshipName() {
      * @return The additional properties defined in the model. This field will contain any properties of the relationship that are not already defined by the other strong types of this class.
```

Comment:
`additional, custom`

Reply:
Because we're removing custom from the name, I want the doc comments to bring out the fact these are not core properties of the object, yet ones they've added, as defined by the model.